language/Russian/strings.po

```diff
@@ -3126,11 +3165,11 @@ msgstr ""
 #: src/frontend/mame/ui/selmenu.cpp:693
 #, c-format
 msgid "Software is clone of: %1$s"
-msgstr "Программа - clone от: %1$s"
+msgstr "Программа - клон от: %1$s"
 
 #: src/frontend/mame/ui/selmenu.cpp:695
 msgid "Software is parent"
-msgstr "Программа - parent"
+msgstr "Программа - оригинал"
```

Comment:
Please don’t translate “parent” in a way that implies “original” – it’s misleading.  Parent/clone relationships are a convenience feature for organisation, and the parent is often not the “original” as such.  The parent is generally the latest version of the most widespread variant.  For example world-wide English language releases are usually the parent when the original is Japanese language software.